tests/validation/tests/rke/common.py

```diff
@@ -133,20 +134,30 @@ def wait_for_etcd_cluster_health(node, etcd_private_ip=False):
     endpoints = "127.0.0.1"
     if etcd_private_ip:
         endpoints = node.private_ip_address
-    etcd_tls_cmd = (
-        'ETCDCTL_API=2 etcdctl --endpoints "https://'+endpoints+':2379" '
-        ' --ca-file /etc/kubernetes/ssl/kube-ca.pem --cert-file '
-        ' $ETCDCTL_CERT --key-file '
-        ' $ETCDCTL_KEY cluster-health')
+    if k8s_rancher_version <= "1.21.8":
```

Comment:
is 1.21.8 the break point of the API change ? Or the reason of hardcoding the version here is due to it being the latest of the 1.21.x version line?

I would set this version in a variable next to the `k8s_rancher_version` instead of here.

Also comparing version strings is a bit risky could we use the `packaging.version.parse`  I think there are a few examples in other tests.

Reply:
Yeah 1.21.8 would not be the correct version for comparison. As 1.22 k8s version is the break point of API change, i'm changing this version from 1.21.x to 1.21.